src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import {
+  Accordion,
+  Loading,
+  NoValue,
+} from '@folio/stripes/components';
+import {
+  AmountWithCurrencyField,
+  FolioFormattedDate,
+  FrontendSortingMCL,
+  DESC_DIRECTION,
+} from '@folio/stripes-acq-components';
+
+import { ACCORDION_ID } from '../const';
+import { useConnectedInvoiceLines } from './useConnectedInvoiceLines';
+
+const COLUMN_INVOICE_DATE = 'invoiceDate';
+const visibleColumns = ['invoiceLine', COLUMN_INVOICE_DATE, 'vendorName', 'vendorInvoiceNo', 'status', 'quantity', 'amount', 'comment'];
+const columnMapping = {
+  invoiceLine: <FormattedMessage id="ui-orders.relatedInvoiceLines.invoiceLine" />,
+  [COLUMN_INVOICE_DATE]: <FormattedMessage id="ui-orders.relatedInvoiceLines.invoiceDate" />,
+  vendorName: <FormattedMessage id="ui-orders.relatedInvoiceLines.vendorName" />,
+  vendorInvoiceNo: <FormattedMessage id="ui-orders.relatedInvoiceLines.vendorInvoiceNo" />,
+  status: <FormattedMessage id="ui-orders.relatedInvoiceLines.status" />,
+  quantity: <FormattedMessage id="ui-orders.relatedInvoiceLines.quantity" />,
+  amount: <FormattedMessage id="ui-orders.relatedInvoiceLines.amount" />,
+  comment: <FormattedMessage id="ui-orders.relatedInvoiceLines.comment" />,
+};
+const resultFormatter = {
+  invoiceLine: invoiceLine => (
+    <Link
+      data-test-link-to-invoice
+      to={`/invoice/view/${invoiceLine.invoice?.id}/line/${invoiceLine.id}/view`}
+    >
+      {`${invoiceLine.invoice?.folioInvoiceNo}-${invoiceLine.invoiceLineNumber}`}
```

Comment:
`undefined-1` and bad link to invoice line if something goes wrong with the invoice fetching

Reply:
something is wrong that invoice line is fetched for removed invoice :) so it shouldn't be an issue